docs/dag-serialization.rst

```diff
@@ -63,6 +63,8 @@ Add the following settings in ``airflow.cfg``:
     If set to True, Webserver reads from DB instead of parsing DAG files
 *   ``min_serialized_dag_update_interval``: This flag sets the minimum interval (in seconds) after which
     the serialized DAG in DB should be updated. This helps in reducing database write rate.
+*   ``store_code``: This flag decides whether to persist DAG files code in DB.
+    If set to True, Webserver reads file contents from DB instead of trying to access files in a DAG folder.
 
 If you are updating Airflow from <1.10.7, please do not forget to run ``airflow db upgrade``.
 
```

Comment:
We can remove the limitation at https://github.com/apache/airflow/pull/7217/files#diff-e2bf1cfc6c93c9cada2dad85c28e4612R79-R80

Reply:
I am afraid I do not understand

Reply:
We can remove Line 79 and 80 about the following limitation:

```
*   **Code View** will read the DAG File & show it using Pygments.
     However, it does not need to Parse the Python file so it is still a small operation.
```